selftest/common/common.sh

```diff
@@ -33,7 +33,7 @@ okcontinue()   { okay  "$1";         }
 kern_version() {
   _oper=$1; _version=$2; _notfatal=$3;
   _given=$(($(echo $_version | sed 's:\.::g')))
-  _current=$(($(uname -r | cut -d'.' -f1,2 | sed 's:\.::g')))
+  _current=$(($(uname -r | cut -d'.' -f1,2,3 | sed 's:\.::g')))
```

Comment:
Updated this check to make it work with my release `6.3.11-200.fc38.x86_64`

Reply:
`echo '6.3.11-200.fc38.x86_64' | cut -d'.' -f1,2 | sed 's:\.::g'`
63

It seems ok, we need to merge only the first and second field of the numeric version.

Could you please recheck this?

Reply:
On https://github.com/aquasecurity/libbpfgo/pull/356

```
[javierhonduco@fedora reuse-fd]$ ./run.sh 
[!] ERROR: kernel 64 not gt than 418
```

(Now with kernel `6.4.6-200.fc38.x86_64`)

Reply:
Hmmm, it seems a logic issue. I'm going to take a look at this later. Thanks.